pkgs/development/compilers/cakelisp/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchFromGitHub, clang, jam }:
```

Comment:
```suggestion
{ stdenv, fetchFromGitHub, clang, jam }:
```

Reply:
This shouldn't be needed because I used lib.optionalString